docker-compose.yml

```diff
@@ -153,3 +155,14 @@ services:
     image: local/tecken_oidcprovider
     ports:
       - "8081:8080"
+
+  # Fake Google Cloud Storage
+  gcs-emulator:
+    image: jwhitlock/gc-fake-storage:test
```

Comment:
This is in your namespace and has a "test" tag. I think we either want to make this more durable now by moving the stuff to a better place or write up a bug to do it soon as a follow-up PR.

The socorro_app is here:

https://hub.docker.com/r/mozilla/socorro_app/

Maybe it makes sense to put it in the same place?

I know with the oidc_testprovider, the fact that it doesn't have a versioned tag creates problems when one tries to update. I think we'd want a versioned tag.

Reply:
I'll have to think about moving the image. My plan is that the upstream project starts officially supporting a dockerized server. I hope that this branched image only lasts another 30 days. That makes me reluctant to start down the path of getting push access to Mozilla's account or delegating creation to ops.

Reply:
That makes sense. Once this lands, we should write up a bug to make sure this gets figured out this summer with some kind of durable solution.

Reply:
Filed as [bug 1563036](https://bugzilla.mozilla.org/show_bug.cgi?id=1563036).